windbow27/Kornblume
# Feat: clear up legacy code

## Desc
1. Correct all the inconsistent item names in the code to avoid confusion in the future.
2. In response to the changes mentioned in point 1, to prevent the warehouse data stored on users' devices from being corrupted, it is necessary to perform a **dirty data clean-up** in `src/App.vue`

## How to test
Just export data from current website, import data on local server and check if the data stored in local storage is expected
